pkg/agent/cniserver/pod_configuration_windows.go

```diff
@@ -134,3 +135,169 @@ func (pc *podConfigurator) reconcileMissingPods(ifConfigs []*interfacestore.Inte
 		}
 	}
 }
+
+type unReadyPodInfo struct {
+	podName      string
+	podNamespace string
+	annotated    bool
+	createTime   time.Time
+}
+
+type podIfaceMonitor struct {
+	kubeClient        clientset.Interface
+	ifaceStore        interfacestore.InterfaceStore
+	ofClient          openflow.Client
+	podUpdateNotifier channel.Notifier
+
+	// unReadyInterfaces is a map to store the OVS ports which is waiting for the PortStatus from OpenFlow switch.
+	// The key in the map is the OVS port name, and its value is unReadyPodInfo.
+	// It is used only on Windows now.
+	unReadyInterfaces sync.Map
```

Comment:
I'm willing to believe that this is a correct usage of `sync.Map`, but I feel like there is no real advantage in using it over a normal `map` + mutex here, and Go discourages its usage.

`updateUnReadyPod` loads values from the map, while `addUnReadyPodInterface` stores values into the map. They are called from different goroutines, and "manage" the same keys. While in theory `updateUnReadyPod` cannot be called concurrently with `addUnReadyPodInterface` (so no race possible), I feel like it makes things harder to reason about?